doc/source/whatsnew/v1.3.4.rst

```diff
@@ -33,10 +33,9 @@ Bug fixes
 
 .. _whatsnew_134.other:
 
-Other
-~~~~~
--
--
+Performance improvements
```

Comment:
this won't make it for 1.3.4, will have to place in 1.4.0. Can add the note to the `Styler` section in that one.

Reply:
done